chapters/annotations.tex

```diff
@@ -887,8 +887,8 @@ \subsubsection{Bitmap}\doublelabel{bitmap}
 When the attribute \lstinline!fileName! is specified, the string refers to an
 external file containing image data. The mapping from the string to the
 file is specified for some URIs in \autoref{external-resources}. The supported file
-formats include \lstinline!PNG!, \lstinline!BMP! and \lstinline!JPEG!, other supported file formats are
-unspecified.
+formats include \lstinline!PNG!, \lstinline!BMP!, \lstinline!JPEG!, 
+and \lstinline!SVG! (provided that the outermost svg-element specify the size, i.e. width and height).
```

Comment:
Given the complexity of the SVG format, I think it would be good to be very precise regarding how the size is specified:
> provided that the outermost \lstinline!svg! element has a size determined by its \lstinline!width! and \lstinline!height! attributes

Since absence of `width` and `height` means the same as 100%, I would think that our intention here is also to require that `width` and `height` are set to absolute lengths.

Reply:
Also, what about unitless lengths?  If they should be allowed, I think we need to specify what the _current user coordinate system_ is in this case.

Reply:
We only need to know the ratio of width/height for our purposes

Reply:
Just to be clear width and height of "outermost svg element" is part of the SVG-specification:
http://www.w3.org/TR/SVG11/struct.html#SVGElementHeightAttribute

> height = "<length>"
> For outermost svg elements, the intrinsic height of the SVG document fragment

However, it _seems_ that it can be in percentages (not sure if viewBox is needed for this or not); so I added "absolute size" to clarify that.

Reply:
Sure, I just wanted it to be explicit that we are taking precisely about these attributes, and not just _width_ and _height_ in general.

Reply:
Changed to lstinline for width and height.